scripts/deploy-chart-test.sh

```diff
@@ -105,7 +108,9 @@ down() {
     $VECTOR_TEST_HELM delete --namespace "$NAMESPACE" "$HELM_CHART"
   fi
 
-  $VECTOR_TEST_KUBECTL delete namespace "$NAMESPACE"
+  if $VECTOR_TEST_KUBECTL get namespace "$NAMESPACE" &>/dev/null; then
+    $VECTOR_TEST_KUBECTL delete namespace "$NAMESPACE"
+  fi
```

Comment:
Stops us from failing the test if we try and delete the same namespace twice, which happens in `vector.rs` tests